packages/clay-css/src/scss/components/_stickers.scss

```diff
@@ -18,12 +18,6 @@
 	text-align: center;
 	vertical-align: middle;
 
-	.lexicon-icon {
```

Comment:
hey @pat270 removing it means we will no longer have icons in Sticker?

Reply:
@matuzalemsteles This broke icon alignment in stickers in IE11. We still have icons in stickers.